src/core/logger.c

```diff
@@ -5,35 +5,56 @@
 #include <string.h>
 #include <time.h>
 
-static int NUM_LEVELS = 6;
-
-static const char *loglevels[] = {
```

Comment:
Hm, sorry, I didn't realize the array was already there.

Reply:
Yeah, that was my addition a while ago to avoid using printf everywhere. I've got no reservations about improving it.

Reply:
So should we use the hardcoded one, or the array solution? I'm tending more to the array version and reverting the last commit, but in the end I don't care that much. Your choice.

Reply:
The array does feel a bit more fragile than what you have now. I'll defer on this one.